app/routers/composite_router.py

```
@@ -0,0 +1,41 @@
+from ..models import VoiceComposite
+
+router = APIRouter(tags=["composite"])
+
+
+@router.post("/voices/{voice_id}/composite")
+async def recompute_voice_composite(voice_id: int):
+    db = next(get_db())
+    service = CompositeService(db)
+    try:
+        return service.compute_and_save_composite(voice_id)
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)